api/v2/checluster_types.go

```diff
@@ -137,6 +138,16 @@ type CheClusterDevEnvironments struct {
 	// +kubebuilder:validation:Minimum:=1
 	// +kubebuilder:default:=300
 	StartTimeoutSeconds *int32 `json:"startTimeoutSeconds,omitempty"`
+	// DeploymentStrategy defines the deployment strategy to use to replace existing workspace pods
+	// with new ones. The available deployment stragies are "Recreate" and "RollingUpdate".
+	// With the "Recreate" deployment strategy, the existing workspace pod is killed before the new one is created.
+	// With the "RollingUpdate" deployment strategy, a new workspace pod is created and the existing workspace pod is deleted
+	// only when the new workspace pod is in a ready state.
+	// If not specified, the default "Recreate" deployment strategy is used.
+	// +optional
+	// +kubebuilder:default:="Recreate"
+	// +kubebuilder:validation:Enum=Recreate;RollingUpdate
+	DeploymentStrategy appsv1.DeploymentStrategyType `json:"deploymentStrategy,omitempty"`
```

Comment:
Please use `` instead of "" in comments. 
It is mostly about how comments are converted into che-docs.

Reply:
Will do :+1:

Reply:
Fixed